prime-router/settings/organizations.yml

```diff
@@ -358,6 +358,21 @@
             redox_source_name: "Prime Data Hub (Staging)"
```

Comment:
We should combine organizations.yml with organizations-local.yml.   Its not worth the trouble having the distinction.